src/components/atoms/InfiniteScroll/hooks.ts

```diff
@@ -0,0 +1,23 @@
+import { MutableRefObject } from "react";
+
+export default (onLoadMore?: () => void) => {
+  const handleAutoFillPage = (ref: MutableRefObject<HTMLDivElement | undefined | null>) => {
+    if (ref.current && ref.current?.scrollHeight <= document.documentElement.clientHeight) {
+      onLoadMore?.();
+    }
```

Comment:
I think something about this isn't quite working correctly. In the Asset settings page you get this called when the page loads, before any scrolling. It isn't a BIG deal, since it only seems to call once, but it really shouldn't call UNLESS the page is too long for the initial fetch.

Reply:
Just happen to see this. I think this part comes from my suggestion some weeks ago. This is only for solving large screen issue (first fetch cannot fill the area) and yes it should be called only under this situation but the point is it seems to be some difficult to check if it is too long (from my memory it was due to the DOM structure is different with different usecases and the function is a common util at that time). So my idea was always want the content fill the area and if not and if there's more data then load more. At that time i only look into this with load projects in large screen issue and i'm not sure about other situations. And also i think the code changes a lot now maybe can optimize this with adding an area's ref?